cli.js

```diff
@@ -42,6 +42,8 @@ const handleCommand = async function handleCommand(options, specificHandler) {
 
 const runtimes = require('./lib/runtimes');
 
+const executionModels = ['exclusive', 'concurrent'];
```

Comment:
this should be derived from the conf type

Reply:
This should be derived from the what?